ludwig/models/llm.py

```diff
@@ -196,6 +212,15 @@ def initialize_adapter(self):
     def to_device(self, device):
```

Comment:
pretty sure `to_device` isn't called when using the deepspeed trainer

Reply:
Agreed